WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift

```diff
@@ -294,3 +352,35 @@ extension UIColor {
         }
     }
 }
+
+extension UIColor {
+    // A way to create dynamic colors that's compatible with iOS 11 & 12
+    convenience init(light: UIColor, dark: UIColor) {
+        #if XCODE11
+        if #available(iOS 13, *) {
+            self.init { traitCollection in
+                if traitCollection.userInterfaceStyle == .dark {
+                    return dark
+                } else {
+                    return light
+                }
+            }
+        } else {
+            self.init(color: light)
+        }
+        #else
+            // in older versions of iOS, we assume light mode
+            self.init(color: light)
+        #endif
```

Comment:
I think this can be simplified by removing the 2 _else_ case. You only need to check if it's _XCODE11_ and _iOS 13_. In all the other cases do the same thing.
```
#if XCODE11
        if #available(iOS 13, *) {
            self.init { traitCollection in
                if traitCollection.userInterfaceStyle == .dark {
                    return dark
                } else {
                    return light
                }
            }
        }
#endif
self.init(color: light)
```
You can simplify this also in the colors properties above.

Reply:
Ah ya, I like that a lot better!

Reply:
I wasn't able to use this pattern for `init(light: UIColor, dark: UIColor)` because `self.init` doesn't return, however, it looks great for all the color properties.